### PR TITLE
OCP: Add SI-6(d) file_integrity_notification_enabled

### PR DESCRIPTION
**Pull request by Vincent056**
SI-6(d) requires the configuration and support of alerts by our operators, This PR adds rule `file_integrity_notification_enabled` to check if a Prometheus rule object is created by File Integrity Operator.

### CODE REVIEW

File: applications/openshift/general/file_integrity_notification_enabled/tests/ocp4/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+default_result: FAIL
+result_after_remediation: PASS
```
Comment (JAORMX) [inline applications/openshift/general/file_integrity_notification_enabled/tests/ocp4/e2e.yml:3]: nice to see this passes. Where's the remediation though? I'm guessing this relies on the manual remediation we already have that installs FIO, does it?

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: applications/openshift/general/file_integrity_notification_enabled/tests/ocp4/e2e.yml (hunk 1 above)
Comment (Vincent056) [reply applications/openshift/general/file_integrity_notification_enabled/tests/ocp4/e2e.yml:3]: Yeah it does, `applications/openshift/integrity/file_integrity_exists/tests/ocp4/e2e-remediation.sh`

File: controls/nist_ocp4.yml [hunk 2]
```diff
@@ -15802,6 +15802,14 @@ controls:
 
     https://issues.redhat.com/browse/CMP-544
   rules: []
+- id: SI-6(d)
```
Comment (JAORMX) [inline controls/nist_ocp4.yml:15805]: This is adding a new control (`SI-6(d)`) and not adding it to any level. So this won't get included in any profile. Let's add it to SI-6, the (d) section is not really a control but merely a bullet point in the control's verbiage.

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: controls/nist_ocp4.yml (hunk 2 above)
Comment (Vincent056) [reply controls/nist_ocp4.yml]: Just fixed it

**Review by JAORMX — APPROVED** [review]
LGTM. I'll merge it once CI passes

**Pull request merged**